symfony/plugins/orangehrmAttendancePlugin/Api/EmployeeAttendanceRecordAPI.php

```diff
@@ -136,7 +137,10 @@ public function getValidationRuleForGetAll(): ParamRuleCollection
             $this->getValidationDecorator()->requiredParamRule(
                 new ParamRule(
                     CommonParams::PARAMETER_EMP_NUMBER,
-                    new Rule(Rules::IN_ACCESSIBLE_EMP_NUMBERS, [false])
+                    new Rule(Rules::POSITIVE),
+                    new Rule(Rules::ENTITY_ID_EXISTS, [Employee::class]),
+                    new Rule(EmployeeAccessibleRule::class, ['apiv2_attendance_employee_attendance_record']),
```

Comment:
I added like this for re using purpose, since this is needed in Employee attendance summary as well.